test/utils/browser.js

```diff
@@ -1,23 +1,35 @@
 // Load in dependencies
 var assert = require('assert');
+var asserters = require('wd/lib/asserters');
+var async = require('async');
 var fs = require('fs');
 var functionToString = require('function-to-string');
-var async = require('async');
 var wd = require('wd');
 
-// Resolve cookies with helpful messaging
-var cookiesJson;
-try {
-  cookiesJson = fs.readFileSync(__dirname + '/../cookies.json');
-} catch (err) {
-  throw new Error('Could not read `test/cookies.json`. Please make sure it exists. ' +
-      'If it doesn\'t, follow the steps in https://github.com/twolfson/google-music.js#testing');
-}
-var cookies = JSON.parse(cookiesJson);
-
 // Resolve the compiled script
 var script = fs.readFileSync(__dirname + '/../../dist/google-music.js', 'utf8');
 
+// Extract Google Music email/password from environment variables
+var GOOGLE_MUSIC_JS_EMAIL = process.env.GOOGLE_MUSIC_JS_EMAIL;
+var GOOGLE_MUSIC_JS_PASSWORD = process.env.GOOGLE_MUSIC_JS_PASSWORD;
+
+assert(
+  GOOGLE_MUSIC_JS_EMAIL,
+  'Username for Google Music wasn\'t set. Please set it via the GOOGLE_MUSIC_JS_EMAIL environment variable'
+);
+assert(
+  GOOGLE_MUSIC_JS_PASSWORD,
+  'Password for Google Music wasn\'t set. Please set it via the GOOGLE_MUSIC_JS_PASSWORD environment variable'
+);
+
+// Extract BrowserStack credentials from environment variables
+// DEV: These are stored securely in Travis CI. Please never ever commit/push them
+var BS_USER = process.env.BS_USER;
```

Comment:
Can we use the expanded words `BROWSERSTACK_USER` and `BROWSERSTACK_KEY`? I'm sure our editors have tab completion which will work for typing these quickly =/